CMakeLists.txt.user

```diff
@@ -0,0 +1,487 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should go in gitignore probably?

Reply:
I deleted it ;-)